.github/CONTRIBUTING.md

```diff
@@ -438,6 +438,16 @@ yarn lint
 yarn format
 ```
 
+#### Developing API endpoints
+
+API endpoints are written in controllers, which are located in `packages/backend/src/controllers`. Controllers are 
+then registered in `packages/backend/src/index.ts` but in order to be made available you'll need to regenerate the 
+`routes.ts` file by executing:
+
+```shell
+yarn workspace backend run tsoa routes

```

Comment:
we should probably have that running during the `yarn dev`. they must have a watch mode..

Reply:
Yes! Can we move to another PR? I have some questions about how `ts-node-dev` works

Reply:
yep, let's do it in a separate PR